src/action/AsyncUserActions.ts

```diff
@@ -213,7 +213,7 @@ export function createNewUser(user: UserAccountData) {
   const action = { type: ActionType.CREATE_USER };
   return (dispatch: ThunkDispatch) => {
     dispatch(asyncActionRequest(action));
-    return Ajax.post(
+    return Ajax.put(
```

Comment:
See my comment on the corresponding backend PR - I don't understand why you changed the HTTP method from POST (which is supposed to be used for creating resources) to PUT (which is supposed to be used for updating resources).